Makefile

```diff
@@ -55,10 +55,10 @@ lint: metadeps  ## Run all quality checks / linting / reformatting
 # Testing
 # -------
 
-tests: venv install metadeps  ## Run tests on the Python package (not including optional dependencies)
+tests: install metadeps  ## Run tests on the Python package (not including optional dependencies)
 	$(VENV_BIN)/pytest tests -m "not optional_dependency"
 
-tests-dep: venv install-all metadeps ## Run tests on the Python package only for optional dependencies
```

Comment:
Calling `venv` deletes my virtual environment. I'd never want that while testing.

Reply:
This is intentional as I want to test in a clean environment; however, I can understand how that is frustrating in active development. ` build-full` probably satisfies my need for now. So feel free to remove `venv` from the two tests targets.